<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adc_demoClass</class>
 <widget class="QDialog" name="adc_demoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>adc_demo</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>800</width>
     <height>110</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_19">
    <attribute name="title">
     <string>Chnl</string>
    </attribute>
    <widget class="QLabel" name="label_wave1Chnl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>chnl1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_wave2Chnl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>chnl2:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_chnl1">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>11</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_chnl2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_chnl1Amp">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>rms:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_chnl2Amp">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>rms:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_chnl1Angle">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>angle:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_chnl2Angle">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>40</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>angle:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_20">
    <attribute name="title">
     <string>Source</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_outputSwitch">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>14</y>
       <width>89</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Source</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_changeSource">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>14</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>toDC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Vol:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>14</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add1">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+1V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_minus1">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-1V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add0p1">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+0.1V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_minus0p1">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-0.1V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add0p01">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+0.01V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_minus0p01">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-0.01V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_minus5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-5V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+5V</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_21">
    <attribute name="title">
     <string>Range</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_chnl2SetRange">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Factor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chnl1SetRange">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Factor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>15</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>k:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chnl2B">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>45</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chnl1B">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>k:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_wave2Chnl_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>chnl2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_chnl1Range">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_chnl2Range">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chnl1K">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chnl2K">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>45</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>15</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_wave1Chnl_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>chnl1:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_22">
    <attribute name="title">
     <string>PWM Ctrl</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>16</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PWM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pwm">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>113</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>VPWM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_vpwm">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>14</y>
       <width>113</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>SPWM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_spwm">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>14</y>
       <width>113</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_setPercent">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Percent</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_23">
    <attribute name="title">
     <string>Record</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_record">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>14</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Record</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_24">
    <attribute name="title">
     <string>Exit</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_close">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
